firmware/ec/.clang-format

```diff
@@ -0,0 +1,108 @@
+# clang-format configuration file. Intended for clang-format >= 4.
+#
+# Modified from linux kernel .clang-format file. 
+#
+# For more information, see:
+#
+#   Documentation/process/clang-format.rst
+#   https://clang.llvm.org/docs/ClangFormat.html
+#   https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+#
+---
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: true
+AlignOperands: true
+AlignTrailingComments: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: false
+BinPackArguments: true
+BinPackParameters: true
+BraceWrapping:
+  AfterClass: false
+  AfterControlStatement: false
+  AfterEnum: false
```

Comment:
As per our coding [guidelines](https://github.com/Telecominfraproject/OpenCellular/commit/31d75a0ffdb4ba0b8a003854f8e529e9843c4ff3#diff-7352ce259d29fe5b59aec6be93bf96adR50) , i think it should be true or have BreakBeforeBraces as GNU so that the enums are not defined in one line .

Reply:
If this was set to true, then the guidelines would have:
```
typedef enum SbdFlowControl 
{
SBD_FLOW_CONTROL_DISABLED = 0,
```
and not have it like it does now. 
```
typedef enum SbdFlowControl {
SBD_FLOW_CONTROL_DISABLED = 0,
```
I'll look into what setting there is to make enum not be defined on one line, though from what I'm seeing there doesn't seem to be an easy way to do that with clang-format :-/